cmd/atlas/atlas.go

```diff
@@ -58,8 +58,9 @@ func loadConfig() error {
 }
 
 func shouldCopyConfig(atlasConfigPath string) bool {
-	// Keep backward compatibility and copy if non-tty
-	if !terminal.IsTerminal(os.Stdout) {
+	// Keep backward compatibility and copy if non-tty. If any shows as non-tty, then we can't ask
+	// questions.
+	if !(terminal.IsTerminal(os.Stdout) && terminal.IsTerminal(os.Stderr) && terminal.IsTerminalInput(os.Stdin)) {
```

Comment:
should this be `or`?

Reply:
either
```golang
if !(terminal.IsTerminal(os.Stdout) && terminal.IsTerminal(os.Stderr) && terminal.IsTerminalInput(os.Stdin))
```
or
```golang
if !terminal.IsTerminal(os.Stdout) || !terminal.IsTerminal(os.Stderr) || !terminal.IsTerminalInput(os.Stdin))
```` 

works, we can use or to make it less expensive